config/crd/bases/idmocp.redhat.com_idms.yaml

```diff
@@ -1,6 +1,6 @@
 
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
This is one of the effects related with 'CRD_OPTIONS ?= "crd:trivialVersions=true, crdVersions=v1"' into the Makefile.

This is generated by `controller-gen` from the `api/` directory.

Reply:
Recovering some notes; this is evoked by changes to remove the following warning.

`Warning: apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition`